src/plugins/controls/connectivity/pauseOnError.js

```diff
@@ -52,13 +52,17 @@ export default pluginFactory({
         const testRunner = this.getTestRunner();
         const returnToHome = () => testRunner.trigger('pause', pauseContext);
         const reloadPage = () => testRunner.trigger('reloadpage');
-        const processError = () => {
+        const processError = (error) => {
```

Comment:
**chore:** The coding style convention says: no parenthesis around single parameters for arrow functions